google-cloud-tools-plugin/resources/messages/CloudToolsBundle.properties

```diff
@@ -16,6 +16,23 @@
 
 select.project.signin=Sign in with your Google account to list your Google Developers Console projects.
 
+cloud.project.label=Project\:
+
+appengine.cloudsdk.location.label=Cloud SDK path\:
+appengine.cloudsdk.location.browse.button=Browse to Cloud SDK location
```

Comment:
Usually in these circumstances the button just says "Browse". The above label says what the user is browsing for.